packages/utils/src/wkg.rs

```diff
@@ -46,7 +46,7 @@ url = "http://localhost:8090"
     }
 
     /// Helper function to initialize a client with the appropriate domain
-    async fn get_client(&self, domain: Option<&String>) -> Result<CachingClient<FileCache>> {
+    async fn get_client(&self, domain: Option<&String>) -> CachingClient<FileCache> {
```

Comment:
I think we shouldn't panic here, just propagate the error, since this can be used lazily in wavs nodes

Reply:
[0886554](https://github.com/Lay3rLabs/WAVS/pull/1069/commits/08865540ecbe0dad5abf5fbb33512aa1f360a02b)